Description: Adjusts the dependencies for OpenJPA
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -763,6 +763,7 @@
 			exclude group: 'org.apache.geronimo.specs', module: 'geronimo-jta_1.1_spec'
 			exclude group: 'org.apache.geronimo.specs', module: 'geronimo-jms_1.1_spec'
 		}
+		optional("org.apache.openjpa:openjpa-persistence-jdbc:${openjpaVersion}")
 		optional("javax.jdo:jdo-api:3.1") {
 			exclude group: 'javax.transaction', module: 'transaction-api'
 		}
