Description: Disable the Caffeine support (not in Debian)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -670,6 +670,14 @@
 		testRuntime("org.ehcache:ehcache:${ehcache3Version}")
 		testRuntime("org.terracotta:management-model:2.3.0")
 	}
+
+	sourceSets {
+		main {
+			java {
+				exclude "**/caffeine/*"
+			}
+		}
+	}
 }
 
 project("spring-web") {
